include/mega/types.h

```diff
@@ -440,7 +440,8 @@ typedef enum {
     ATTR_PWD_REMINDER = 15,     // private, non-encrypted - char array in B64 - non-versioned
     ATTR_DISABLE_VERSIONS = 16, // private, non-encrypted - char array in B64 - non-versioned
     ATTR_CONTACT_LINK_VERIFICATION = 17,  // private, non-encrypted - char array in B64 - non-versioned
-    ATTR_RICH_PREVIEWS = 18      // private - byte array
+    ATTR_RICH_PREVIEWS = 18,     // private - byte array
+    ATTR_LAST_PSA = 19      // private - char array
```

Comment:
We will need to adjust types for attributes. The 19 is also used in the https://github.com/meganz/sdk/pull/1190 for the RubbishTime.